scripts/taxonomy/classification.py

```diff
@@ -108,21 +132,29 @@ def main(ctx, db_url: str, solr_url: str, max_records: int, verbosity: str):
     specimen_mapping = getHierarchyMapping("specimen")
 
     for thing in thing_iterator.yieldRecordsByPage():
-        print(f"thing is {thing.id}")
+        # print(f"thing is {thing.id}")
         transformed = SESARTransformer.SESARTransformer(
             thing.resolved_content
         ).transform()
 
         # extract text and gold label from object
-        text, gold_material, gold_sample = parse_SESAR_thing(
-            thing.resolved_content
+        (
+            description_map,
+            material_text, sample_text,
+            gold_material, gold_sample
+        ) = parse_SESAR_thing(thing.resolved_content)
```

Comment:
If these are common fields that belong together, it might make sense to refactor this to a class.  Grouped tuples are fine for work in progress but we should formalize it if we're going to use this group of fields together more than for debugging.